lusSTR/wrappers/convert.py

```diff
@@ -170,6 +175,48 @@ def sort_table(table):
     return sorted_table
 
 
+def marker_plots(df, output_name, sex):
+    Path("MarkerPlots").mkdir(parents=True, exist_ok=True)
+    df["CE_Allele"] = df["CE_Allele"].astype(float)
+    for id in df["SampleID"].unique():
+        sample_id = f"{id}_sexchr" if sex else id
+        with PdfPages(f"MarkerPlots/{output_name}_{sample_id}_marker_plots.pdf") as pdf:
+            make_plot(df, id, sex, False)
+            pdf.savefig()
+            make_plot(df, id, sex, True)
+            pdf.savefig()
```

Comment:
That way, when someone is messing with the code 18 months from now and they're trying to remember what that dang 4th argument to the `make_plot` function is, passing the argument as a key/value pair will provide a great contextual reminder.

```python
make_plot(df, id, sex, sameyaxis=False)
```